plugins/search-react/src/components/SearchFilter/SearchFilter.tsx

```diff
@@ -188,13 +213,37 @@ export const SelectFilter = (props: SearchFilterComponentProps) => {
         </InputLabel>
       ) : null}
       <Select
+        className={classnames({
```

Comment:
maybe I missed sth, but I didn't find how to change styles when any value is selected, e.g. I want to highlight border and color, mui doesn't add any class, so I did it

Reply:
I wonder if you can style the menu items with something like this instead: 
```tsx
menuProps={{ classes: { selected: filters[name] }}
```

Reply:
(and you have menuProps below)

Reply:
instead - no, because menu is a popup which is adding to a dom in an absolutely different place and I need to manipulate styles of an input itself.

Can we add `selected` class for the menu also ? I think yes, just didn't see any need of that as generally for menus we highlight only a selected item, not the entire menu